contracts/voting-verifier/src/query.rs

```diff
@@ -1,58 +1,37 @@
-use axelar_wasm_std::operators::Operators;
-use axelar_wasm_std::voting::{PollStatus, Vote};
+use axelar_wasm_std::{
+    operators::Operators,
+    voting::{PollStatus, Vote},
+    VerificationStatus,
+};
 use connection_router::state::{CrossChainId, Message};
-use cosmwasm_schema::cw_serde;
 use cosmwasm_std::Deps;
 
 use crate::error::ContractError;
 use crate::state::{self, Poll, PollContent, POLLS, POLL_MESSAGES, POLL_WORKER_SETS};
 
-#[cw_serde]
-pub enum VerificationStatus {
-    Verified,
-    FailedToVerify,
-    InProgress,  // still in an open poll
-    NotVerified, // not in a poll
-}
-
-pub fn is_verified(
+pub fn messages_status(
     deps: Deps,
     messages: &[Message],
-) -> Result<Vec<(CrossChainId, bool)>, ContractError> {
+) -> Result<Vec<(CrossChainId, VerificationStatus)>, ContractError> {
     messages
         .iter()
-        .map(|message| {
-            msg_verification_status(deps, message).map(|status| {
-                (
-                    message.cc_id.to_owned(),
-                    matches!(status, VerificationStatus::Verified),
-                )
-            })
-        })
+        .map(|message| msg_status(deps, message).map(|status| (message.cc_id.to_owned(), status)))
         .collect::<Result<Vec<_>, _>>()
 }
 
-pub fn is_worker_set_verified(deps: Deps, operators: &Operators) -> Result<bool, ContractError> {
-    Ok(matches!(
-        worker_set_verification_status(deps, operators)?,
-        VerificationStatus::Verified
-    ))
-}
-
-pub fn msg_verification_status(
-    deps: Deps,
-    message: &Message,
-) -> Result<VerificationStatus, ContractError> {
+pub fn msg_status(deps: Deps, message: &Message) -> Result<VerificationStatus, ContractError> {
```

Comment:
Don't mix `msg` and `message`.
```suggestion
pub fn message_status(deps: Deps, message: &Message) -> Result<VerificationStatus, ContractError> {
```